float64 x_mid
float64 y_mid
float64 box_size
int32 box_count